### PR TITLE
Moving the UI part to a separate subspec

### PR DESCRIPTION
**Pull request by hfossli**
Keeping both Core and UI as default subspecs

### CODE REVIEW

**Comment by hfossli** [comment]
The `Contentful` subspec emits some warnings, but I don't think they got introduced now. 
```
➜  Concorde git:(master) ✗ pod lib lint

 -> Concorde (0.1.1)
    - WARN  | [Concorde/Contentful] xcodebuild:  AFNetworking/AFNetworking/AFNetworkReachabilityManager.m:111:17: warning: method override for the designated initializer of the superclass '-init' not found [-Wobjc-designated-initializers]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/usr/include/objc/NSObject.h:60:1: note: method marked as designated initializer of the class here
    - WARN  | [Concorde/Contentful] xcodebuild:  AFNetworking/AFNetworking/AFURLConnectionOperation.m:159:17: warning: method override for the designated initializer of the superclass '-init' not found [-Wobjc-designated-initializers]
    - WARN  | [Concorde/Contentful] xcodebuild:  AFNetworking/AFNetworking/AFURLSessionManager.m:319:62: warning: null passed to a callee that requires a non-null argument [-Wnonnull]
    - WARN  | [Concorde/Contentful] xcodebuild:  AFNetworking/AFNetworking/AFURLSessionManager.m:319:92: warning: null passed to a callee that requires a non-null argument [-Wnonnull]
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:85:73: warning: 'NSGregorianCalendar' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarIdentifierGregorian instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSLocale.h:115:36: note: 'NSGregorianCalendar' has been explicitly marked deprecated here
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:216:65: warning: 'NSYearCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitYear instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSCalendar.h:65:2: note: 'NSYearCalendarUnit' has been explicitly marked deprecated here
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:216:86: warning: 'NSMonthCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitMonth instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSCalendar.h:66:2: note: 'NSMonthCalendarUnit' has been explicitly marked deprecated here
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:216:108: warning: 'NSDayCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitDay instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSCalendar.h:67:2: note: 'NSDayCalendarUnit' has been explicitly marked deprecated here
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:817:63: warning: 'NSYearCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitYear instead [-Wdeprecated-declarations]
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:817:84: warning: 'NSWeekdayCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitWeekday instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSCalendar.h:72:2: note: 'NSWeekdayCalendarUnit' has been explicitly marked deprecated here
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:817:108: warning: 'NSDayCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitDay instead [-Wdeprecated-declarations]
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:820:74: warning: 'NSYearCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitYear instead [-Wdeprecated-declarations]
    - WARN  | [Concorde/Contentful] xcodebuild:  ISO8601DateFormatter/ISO8601DateFormatter.m:823:70: warning: 'NSDayCalendarUnit' is deprecated: first deprecated in iOS 8.0 - Use NSCalendarUnitDay instead [-Wdeprecated-declarations]
    - NOTE  | [Concorde/Contentful] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/objc/NSObject.h:60:1: note: method marked as designated initializer of the class here

[!] Concorde did not pass validation, due to 13 warnings (but you can use `--allow-warnings` to ignore them).
```

File: Concorde.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Concorde"
-  s.version          = "0.1.0"
+  s.version          = "0.1.1"
```
Comment (neonichu) [inline Concorde.podspec]: Could you remove this change? I prefer doing the version bump as its own separate change later.

**Comment by neonichu** [comment]
LGTM except for the one comment, thanks for the PR! The warnings were indeed there already.

File: Concorde.podspec (hunk 1 above)
Comment (hfossli) [inline Concorde.podspec]: Allright, I didn't know podspecs could be updated therefore I created a new version.

**Comment by hfossli** [comment]
Even though the version number is unchanged, are you still able to publish the new version of the podspec? We don't support swift yet on our project you see.

**Comment by neonichu** [comment]
No, but I'll update the version number as part of my normal release workflow after this has been merged.

**Comment by hfossli** [comment]
Squashed

**Comment by neonichu** [comment]
Awesome, thanks for the contribution!

**Pull request merged**